README.md

```diff
@@ -1,50 +1,24 @@
-NetSquid-SnippetTemplate
-========================
+How to create a NetSquid Snippet
+================================
 
-This repository provides a template for creating [NetSquid snippets](https://netsquid.org/snippets).
-For instructions of how to create a snippet package using this template see the [HOWTO file](HOWTO-SnippetTemplate.md).
+NetSquid snippets should follow the provided [template git repository](https://github.com/SoftwareQuTech/NetSquid-SnippetTemplate).
+The template makes use of [cookiecutter](https://cookiecutter.readthedocs.io/en/latest/index.html).
 
-**TEMPLATE:** remove the above notice and change the title to your snippet's name.
+To instantiated a new folder for a new snippet, clone this repo, cd in to it and do
 
-Description
------------
+```shell
+cookiecutter . -o /your/path
+```
 
-This is a user contributed _snippet_ for the [NetSquid quantum network simulator](https://netsquid.org).
+where `/your/path` is the folder you want to place the new snippet in. Note that you need `cookiecutter` installed which can be done by
 
-**TEMPLATE**: briefly describe what your snippet package does.
+```shell
+pip3 install cookiecutter
+```
```

Comment:
Clearer to introduce and give install instructions for cookiecutter before the first code block that uses it?

I would also be more explicit that the output path is the directory (not folder, see below:)) that the snippet directory will be created in, instead of the snippet project directory itself

Reply:
Done